smithy-swift-codegen/src/test/kotlin/mocks/MockHttpEC2QueryProtocolGenerator.kt

```diff
@@ -108,6 +108,14 @@ class MockHttpEC2QueryProtocolGenerator : HttpBindingProtocolGenerator() {
         // Intentionally empty
     }
 
+    override fun generateMessageMarshallable(ctx: ProtocolGenerator.GenerationContext) {
+        // Intentionally empty
+    }
+
+    override fun generateMessageUnmarshallable(ctx: ProtocolGenerator.GenerationContext) {
+        // Intentionally empty
```

Comment:
Similar to an above question, why is it intentionally empty?

Reply:
something to cover in sync